corehq/apps/sofabed/migrations/0002_delete_models.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
It will definitely cause a huge IO spike. Not sure how long it will take though. Could delete the data manually first via a script (in chunks).